src/views/Deposit/Deposit.js

```diff
@@ -90,47 +80,32 @@ const Deposit = () => {
   const { register, handleSubmit, watch /* , errors  */ } = useForm();
   let amount = watch('amount') ? watch('amount') : 0;
   let balance = Number(web3Connect.daiBalance);
-  // let deposit = Number(web3Connect.daiDeposit);
-  // console.log({ allowance: web3Connect.daiBalance, amount });
   const onSubmit = async data => {
     let { amount } = data;
-
     setStatus(`DEPOSITING`);
-    // const { title, description } = data;
-    console.log(data);
-
-    // let body = { ...data, image };
-
-    // let hash = await uploadJson(data.title, body);
-    // setStatus('SUBMITTING_BLOCKCHAIN');
-
-    // // console.log({ hash });
-    // // let json = await getJson(hash);
-    // // console.log({ json });
     await web3Connect.contracts.dao.methods.triggerDeposit(amount);
```

Comment:
There are lots of merge conflicts in this file. I'll sort them out on my side. Get master to an up to date state again :)